### PR TITLE
Alert & Alert img

### PR DESCRIPTION
**Pull request by akbarjorayev**

### CODE REVIEW

File: src/components/Alert.js [hunk 1]
```diff
@@ -1,19 +1,21 @@
-import React from 'react';
```
Comment (nazarovctrl) [inline src/components/Alert.js:1]: ; nega buni ob tashadin?

File: src/components/Alert.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-          <button onClick={onDelete} className="delete_button">O'chirish</button>
+          {children.filter((item) => item.key === 'alert_below_button')[0]}
         </div>
       </div>
     </div>
-  );
+  )
 }
 
-export default Alert;
```
Comment (nazarovctrl) [inline src/components/Alert.js:19]: ; ?

File: src/pages/Profile.js [hunk 3]
```diff
@@ -152,6 +154,15 @@ function Profile() {
         return <div>Ma'lumotlar yuklanmoqda...</div>;
     }
 
+    function fileInputChange() {
+        setShowAlertPic(true)
+    }
+
+    function hideAlertPic() {
+        setShowAlertPic(false)
```
Comment (nazarovctrl) [inline src/pages/Profile.js:162]: ;  qani

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

**Comment by akbarjorayev** [comment]
boldi qoshldi

**Comment by akbarjorayev** [comment]
Tayyor

File: src/components/Navbar.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ import { Box, Button, Avatar, Container, Menu, Tooltip, Grid, Link, IconButton,
 import { Logout, Settings } from '@mui/icons-material';
 import PersonIcon from '@mui/icons-material/Person';
 import NavbarSimple from "../components/NavbarSimple"
```
Comment (nazarovctrl) [inline src/components/Navbar.js:7]: shu import bilan pasidagi importgayam ;

File: src/pages/Profile.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import React, { useRef } from "react"
+import React, { useRef } from "react";
 import "./css/Profile.css"
```
Comment (nazarovctrl) [inline src/pages/Profile.js:2]: bu joyda yam ; hammasiga

File: src/util/Image.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+function getData(img) {
```
Comment (nazarovctrl) [inline src/util/Image.js:1]: ;

**Review by nazarovctrl — CHANGES_REQUESTED** [review]

**Pull request merged**